lib/galaxy/webapps/tool_shed/app.py

```diff
@@ -10,9 +10,11 @@
 from galaxy.openid.providers import OpenIDProviders
 from galaxy.util.dbkeys import GenomeBuilds
 from galaxy.web import security
+from galaxy.config import configure_logging
```

Comment:
@martenson Remember that we now sort the imports! This should go between `from galaxy import tools` and `from galaxy.managers.tags import CommunityTagManager`